api/src/main/java/ai/djl/ndarray/internal/NDFormat.java

```diff
@@ -60,8 +60,9 @@ public static String format(
         if (array.hasGradient()) {
             sb.append(" hasGradient");
         }
+
         if (DEBUG) {
-            return sb.toString();
+            sb.append(" ~>");
```

Comment:
We need return here. IntelliJ trys to evaluate ndarray in debugging mode, which makes IDE super slow.

Reply:
Would it help if we bound the string with shorter limit? Currently it is already bounded. I found that showing the content of an NDArray (even partially) is quite often needed when debugging a model during training. 
It looks like Pycharm IDE enables evaluating the ndarray(tensor) on pytorch engine, too.